.github/workflows/toxicity-check.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Toxicity check'
+on: 
+  issue_comment:
+    types: ['created', 'edited', 'deleted']
+  pull_request_review:
+  pull_request_review_comment:
+  issues:
+    types: ['opened', 'edited', 'deleted', 'closed', 'reopened', 'locked', 'unlocked']
+  pull_request:
+    types: ['opened', 'edited', 'closed', 'reopened', 'locked', 'unlocked']
+
+jobs:
+  hello_world_job:
+    runs-on: ubuntu-latest
+    name: Toxicity check job
+    steps:
+      - name: Toxicity check
+        uses: TheDutchDevil/toxicity-detection-bot@main
```

Comment:
**If** we end up participating, I think we need an explicit git commit to run from, rather than a branch. It would be dangerous to give our GITHUB_TOKEN to arbitrary code from a stranger.

Reply:
Check, good call! Based on what you decide I'll update the bot to a commit that contains the desired configuration of the bot.